app/controllers/concerns/page_cache_module.rb

```diff
@@ -0,0 +1,7 @@
+module PageCacheModule
+  # 以下参考に実装
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)